### ISSUE TITLE
exe binary reported as malware on scanner sites

### ISSUE THREAD

**Issue by Bassquake** [CHUNx3/MetaQuestBitrateRegistryEditor#6]
The compiled binary is reported as having malware when scanned on the virustotal.com as MaxSecure Trojan.Malware.300983.susgen. Also as Virus/Malware!eZyGlphI Malware Webroot SMD Trend Micro on the metadefender.opswat.com.

**Comment by CHUNx3** [comment]
Thanks for letting us know. I don't know why virustotal.com is scanning like that, that executable is exactly what you get when you compile it after receiving the source here, so that scan is a false positive.
I'll check to make sure it's not a false positive.

**Comment by CHUNx3** [comment]
Similar issue: https://answers.microsoft.com/en-us/windows/forum/all/visual-studio-c-script-detected-as-trojan-virus/f5c8962c-0787-4594-8d1e-3ba5475b7892

**Comment by CHUNx3** [comment]
@Bassquake 
I referred to this [link](https://answers.microsoft.com/en-us/windows/forum/all/visual-studio-c-script-detected-as-trojan-virus/f5c8962c-0787-4594-8d1e-3ba5475b7892)'s comment and I tried to rebuild the project. But it didn't work(still considered a virus only in MaxSecure).

According to the following [link](https://digichasers.com/trojan-malware-300983-susgen/), it seems to the false positive. If you have any concern still, I recommend that you clone this repo's source and build it in Visual Studio. You'll get the same execution file. :)
